Drv/Interfaces/docs/sdd.md

```diff
@@ -20,4 +20,4 @@ The `Drv/SpiInterface` is an interface for writing and reading to an SPI device.
 
 ## Drv/TickInterface
 
-The `Drv/TickInterface` is an interface for outputting a system tick along with a RawTime timestamp. The [`Drv::BlockDriver`](../../BlockDriver/BlockDriver.fpp) driver implements this interface.
+The `Drv/TickInterface` is an interface for outputting a system tick along with a RawTime timestamp. The [`Ref::BlockDriver`](../../../Ref/BlockDriver/BlockDriver.fpp) driver is an example of a notional driver that implements this interface.
```

Comment:
It isn't anymore, while it includes the interface, it never calls CycleOut. Point this to LinuxTimer?

Reply:
Changed to `Svc/LinuxTimer` and changed the timer FPP to use the tick interface.